src/pydata_sphinx_theme/assets/scripts/index.js

```diff
@@ -43,23 +43,25 @@ function scrollToActive() {
   );
 
   if (!isNaN(storedScrollTop)) {
+    // If we've got a saved scroll position, just use that
     sidebar.scrollTop = storedScrollTop;
+    console.log("[PST]: Scrolled sidebar using stored browser position...");
   } else {
-    var active_pages = sidebar.querySelectorAll(".active");
-    var offset = 0;
-    var i;
-    for (i = active_pages.length - 1; i > 0; i--) {
-      var active_page = active_pages[i];
-      if (active_page !== undefined) {
-        offset += active_page.offsetTop;
+    // Otherwise, calculate a position to scroll to based on the lowest `active` link
+    var sidebarNav = document.getElementById("bd-docs-nav");
+    var active_pages = sidebarNav.querySelectorAll(".active");
+    if (active_pages.length > 0) {
+      // Use the last active page as the offset since it's the page we're on
+      var latest_active = active_pages[active_pages.length - 1];
```

Comment:
```suggestion
      var latest_active = active_pages.at(-1);
```
Maybe? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at

Reply:
Only question I have is the "blue" square for safari in the compatibility chart - what does that mean?

Reply:
It also looks like there isn't support for safari mobile etc, so can we stick with the current pattern for now until we know there's more support for safari?